### ISSUE TITLE
plugin not detecting projects

### ISSUE THREAD

**Issue by JurgenCruz** [tarn2206/gradle-dependencies-checker#1]
I have a multiproject gradle solution which also includes outside projects. the plugin doesn't detect any project, either inside or outside.

the folder structure is something like this:

```
-  root
  - mainSolution
    - project1
      - layer1
        - build.gradle.kts
      - layer2
        - build.gradle.kts
      - layer3
        - build.gradle.kts
    - project2
      - layer1
        - build.gradle.kts
      - layer2
        - build.gradle.kts
      - layer3
        - build.gradle.kts
    - project3
      - layer1
        - build.gradle.kts
      - layer2
        - build.gradle.kts
      - layer3
        - build.gradle.kts
    - build.gradle.kts
    - settings.gradle.kts
  - engine
    - layer1
      - build.gradle.kts
    - layer2
      - build.gradle.kts
    - layer3
      - build.gradle.kts
    - build.gradle.kts
    - settings.gradle.kts
  - utils
    - library1
      - build.gradle.kts
    - library2
      - build.gradle.kts
    - library3
      - build.gradle.kts
    - build.gradle.kts
    - settings.gradle.kts
```

The settings.gradle.kts file in root\mainSolution\ is something like this:

```kotlin
include("project1:layer1","project1:layer2","project1:layer3","project2:layer1","project2:layer2","project2:layer3","project3:layer1","project3:layer2","project3:layer3")

includeBuild("../utils")
includeBuild("../engine")
```

Both the inside and outside projects depend on the utils libraries. The inside projects can depend on the outside project. and the inside projects depend between themselves.

Thanks.
